### DISCUSSION TITLE
Video on OpenStatus

### DISCUSSION THREAD

**Discussion by elie222** [openstatusHQ/openstatus#274] [Show and tell]
Just released a video covering the code behind OpenStatus :)

https://www.youtube.com/watch?v=PYfSJATE8v8

**Comment by thibaultleouay** [comment]
Thanks for the video @elie222

**Comment by MdTanwer** [comment]
Hi @elie222,

Sir, could you please help me set this up locally on Windows 11 through WSL? I am stuck with setting it up on my local machine.
